sections/performance/block-loop.md

```diff
@@ -0,0 +1,57 @@
+Node handles the Event Loop on a single thread rotating through multiple queues. Operations with high complexity, large json parsing, unsafe regex querieies, and long awaited promises are some of the operations that can cause the event loop to stall. Preventing this is as easy as setting up monitoring for the the Event Loop and resolving the blocker.
```

Comment:
Did you forget the page title? see other 'more info' files, they all contain a title